src/js/manage_wordles_utils.js

```diff
@@ -0,0 +1,126 @@
+            <input type="text" id="question-id-${ uniqueId }" name="question" value="${ wordleOptions.question }" ${ disableInputs ? "disabled" : ""}>
+        </div>
+        <div class="entries-cell">
+            <label for="entries-id-${ uniqueId }">Word Entries</label>
+            <input type="text" id="entries-id-${ uniqueId }" name="entries" value="${ wordleOptions.entries }" ${ disableInputs ? "disabled" : ""}>
+        </div>`;
+
+    htmlTogo += isCreateNew ? `
+        <div class="view-answer-cell">
+            <svg role="presentation" class="view-answer-svg margin-auto">
+                <use xlink:href="#ellipse"></use>
+            </svg>
+            Generating Link
+        </div>
+        <div class="view-wordle-cell">
+            <svg role="presentation" class="view-wordle-svg margin-auto">
+                <use xlink:href="#ellipse"></use>
+            </svg>
+            Generating Link
+        </div>
+        ` : `
+        <div class="view-answer-cell">
+            <label for="view-answer-id-${ uniqueId }">Answers</label>
+            <a id="view-answer-id-${ uniqueId }" class="button view-answer" href="https://${ wordleOptions.branchName + netlifyUrlSuffix }answer/">
+                <svg role="presentation" class="view-answer-svg">
+                    <use xlink:href="#view"></use>
+                </svg>
+                View
+            </a>
+        </div>
+        <div class="view-wordle-cell">
+            <label for="view-wordle-id-${ uniqueId }">Wordle</label>
+            <a id="view-wordle-id-${ uniqueId }" class="button view-wordle" href="https://${ wordleOptions.branchName + netlifyUrlSuffix }wordle/">
+                <svg role="presentation" class="view-wordle-svg">
+                    <use xlink:href="#view"></use>
+                </svg>
+                View
+            </a>
+        </div>
+        `;
+
+    htmlTogo += `
+        <div class="last-modified-cell">
+            <label for="last-modified-id-${ uniqueId }">Last Modified</label>
+            <span id="last-modified-id-${ uniqueId }">${ wordleOptions.lastModifiedTimestamp }</span>
+        </div>
+        <div class="delete-cell">
+            <button id="delete-id-${ uniqueId }" class="delete-button ${ wordleOptions.extraRowClass ? wordleOptions.extraRowClass : "" }" ${ disableInputs ? "disabled" : ""}>
+                <svg role="presentation" class="view-answer-svg">
+                    <use xlink:href="#cross"></use>
+                </svg>
+                Delete
+            </button>
+        </div>
+        <div class="one-status ${ wordleOptions.extraStatusClass ? wordleOptions.extraStatusClass : "" } ${ wordleOptions.statusMsg ? "" : "hidden" }" role="status">${ wordleOptions.statusMsg ? wordleOptions.statusMsg : "" }</div>
+        <input type="hidden" name="branchName" value="${ wordleOptions.branchName }">
+    </div>\n\n`;
+
+    return htmlTogo;
+};
+
+inverted_wordles.findWordleRowByBranchName = function (branchName) {
```

Comment:
Try to push the point where "globalOptions" is resolved (after it is renamed) as high as possible in the call chain - e.g. pass "selectors" as first argument to this function